en-US/New.xml

```diff
@@ -5,6 +5,86 @@
 ]>
 <section id="new">
         <title>New in This Release</title>
+		<formalpara id="new_70">
+		<title>Release 7.0</title>
+            <para>Major update in October 2024 with migration to a new hosting site, and extensive added and updated guidance.</para>
+                </formalpara>   
+            <itemizedlist>
+				<!-- <listitem>
+                                        <para>This guide is now hosted at https://nnn.redhat.com and no longer at https://stylepedia.net/.</para>
+				</listitem>
+				<listitem>
+                                        <para>The GitHub repository for contributions to this guide is now at https://github.com/RedHatTraining/nnnnn and no longer at https://github.com/StyleGuides/WritingStyleGuide.</para>
+				</listitem> -->
+				<listitem>
+                                        <para>Updated references to IBM Style to include a link, which is to a specific topic page where relevant.
+										Each link is denoted with a padlock icon, to show that access is currently restricted to IBM and Red&nbsp;Hat employees.</para>
+				</listitem>
+				<listitem>
+                                        <para>Removed content that was specific to marketing content.
+										Removed references to AP Style.</para>
+				</listitem>
+				<listitem>
+                                        <para>Updated the audience information.
+										Refer to <xref linkend="audience"/>.</para>			
+				</listitem>
+				<listitem>
+                                        <para>Updated the resources information.
+										Refer to <xref linkend="resources"/>.</para>			
+				</listitem>
+				<listitem>
+                                        <para>Updated guidance to use the simpler phrase "go to" rather than "navigate to", for directing readers to a location.
```

Comment:
In this list, what do you think about grouping all of the usage items together? There are quite a few items that relate to usage and definitions, and it might be useful for readers to keep them together in this list.